gobblin-core/src/main/java/gobblin/writer/JdbcWriterBuilder.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.writer;
+
+import gobblin.converter.jdbc.JdbcEntryData;
+import gobblin.converter.jdbc.JdbcEntrySchema;
+
+import java.io.IOException;
+
+import org.apache.avro.Schema;
+
+public class JdbcWriterBuilder extends PartitionAwareDataWriterBuilder<JdbcEntrySchema, JdbcEntryData> {
```

Comment:
I suggest just extending `DataWriterBuilder` instead, we can add the partitioning integration in another PR, as it doesn't seem to be fully done.